src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -50,6 +50,26 @@ const ParentFamilyHome = props => {
       <ParentSidebar />
       <Content>
         <Banner />
+
+        {alertMsg === 1 && (
```

Comment:
Perfect use case for conditional rendering and like that you used Ant design instead of building an alert from scratch.